### ISSUE TITLE
Automatic synchronization timer

### ISSUE THREAD

**Issue by AeonLucid** [erengy/taiga#245]
Hi,

Is the anime list being refreshed automatically too or just the torrents?

I need this because I run taiga on a server with plex installed to automatically download new episodes from anime that I am watching / planning to watch.

**Comment by erengy** [comment]
Hi there,

When automatic synchronization is enabled (from Tools menu), Taiga checks the update queue every five minutes, and sends the list updates to MyAnimeList or Hummingbird. However, it doesn't download a fresh copy of your list, if that's what you mean. In other words, if your list is updated from somewhere else (e.g. the website or another instance of Taiga), the changes won't be synchronized automatically. Currently that only happens at startup, if you have the relevant option enabled.

**Comment by AeonLucid** [comment]
That sucks. 

Should I make something that automatically closes and opens taiga every hour, like a bat file?
Won't that mess up the torrents part?

**Comment by erengy** [comment]
Yeah, that would work. The torrent timer will be reset when you restart the application, but it shouldn't be an issue if you set the intervals accordingly.

**Comment by AeonLucid** [comment]
I am using this bat script now.
```
@echo off
:loop
start "Taiga" "C:\Users\Administrator\AppData\Roaming\Taiga\Taiga.exe"
timeout /t 360
taskkill /f /im "Taiga.exe"
timeout /t 5
goto loop
```

But somehow, this happens http://i.imgur.com/RxAeRyO.png
Does this have something to do with taiga?
